src/ui/components/cards/accordioncardcomponent.js

```diff
@@ -143,13 +143,13 @@ export default class AccordionCardComponent extends Component {
   }
 
   onMount () {
-    const toggleEl = DOM.query(this._container, '.js-yxt-AccordionCard-toggle');
-    const accordionBodyEl = DOM.query(this._container, '.js-yxt-AccordionCard-body');
     const accordionEl = DOM.query(this._container, '.js-yxt-AccordionCard');
```

Comment:
ah really minor nit but can we move `const accordionEl =` inside the if statement? it's not being used outside of it